config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  root to: 'posts#index'
+  resources :posts do
+    resources :comments
```

Comment:
Make sure to use the correct nested path helpers for Comment elsewhere in your app.